components/app/app.component.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import { HomeComponent } from '@components';
+import Image from 'next/image';
+
+const AppComponent = (): JSX.Element => {
+  return (
+    <div>
+      <Image
+        src={require('@assets/images/logo.png')}
+        alt='logo'
+        width={200}
```

Comment:
Is this the best way to set width and height i.e. should it be harded code like this? Are there better ways to do it?